### DISCUSSION TITLE
Chem_ZJU/Basic/%E6%99%AE%E9%80%9A%E5%8C%96%E5%AD%A6%28%E7%94%B2%29/

### DISCUSSION THREAD

**Discussion by giscus** [wing0night/Chem_ZJU#3] [Announcements]
# Chem_ZJU/Basic/%E6%99%AE%E9%80%9A%E5%8C%96%E5%AD%A6%28%E7%94%B2%29/

求化班学习指南

https://wing0night.github.io/Chem_ZJU/Basic/%E6%99%AE%E9%80%9A%E5%8C%96%E5%AD%A6%28%E7%94%B2%29/

**Comment by wing0night** [comment]
。。。
